### PR TITLE
Add data counter to download counter

### PR DESCRIPTION
**Pull request by SenlisO**
Added a data counter object to download counter file.

Verified that the file size is pulled from the download file.  By default, it is referenced in Bytes.  Also tested that the program still runs with the changes and the file is downloaded.

When testing the program, the daily_count.txt and download_count_longterm.txt were not updating.  I feel this may be because of my test environment.

### CODE REVIEW

**Comment by Batcastle** [comment]
Looks like you have an error on Line 92:
```
Traceback (most recent call last):
  File "/usr/lib/python3.12/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/usr/lib/python3.12/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/batcastle/Desktop/download-optimizer/./download.py", line 92, in update_download_count
    data_count = values[1]
                 ~~~~~~^^^
IndexError: list index out of range
```

If you having issues testing how it reads/writes to the count files, edit the sleep time on line 87. Currently it sleeps for 15 minutes. Might wanna cut that down to 5 or 10 seconds or so.

File: download.py [hunk 1]
```diff
@@ -69,32 +69,39 @@ def haversine(point_1, point_2, units="km"):
...
         time.sleep(900)
         with LOCK:
             with open(common.CURRENT_COUNT_FILE, "r") as file:
-                count = file.read()
+                values = file.read().split(',')
+            count = values[0]
+            data_count = values[1]
             try:
                 count = int(count)
+                data_count = int(count)
```
Comment (Batcastle) [inline download.py]: You set `data_count` to `count` here while those should be separate.

File: download.py [hunk 2]
```diff
@@ -69,32 +69,39 @@ def haversine(point_1, point_2, units="km"):
 # Multithreading stuffs
 LOCK = multiprocessing.Lock()
 COUNTER = multiprocessing.RawValue("i", 0)
+DATA_COUNTER = multiprocessing.RawValue("i", 0)
```
Comment (Batcastle) [inline download.py]: Change this from an "int" to an "unsigned long" by changing the "i" to an "L" in the first argument here.

Comment (Batcastle) [reply download.py]: This should fix the integer overflow issue.

**Comment by SenlisO** [comment]
ready for next review

**Comment by Batcastle** [comment]
Looks good! Thank you much!

**Pull request merged**